Makefile

```diff
@@ -64,12 +64,12 @@ template: clean-template library
 	$Dprosv5 c create-template $(TEMPLATE_DIR) kernel $(shell cat $(ROOT)/version) --system "./**/*" --user "src/opcontrol.{c,cpp,cc}" --user "src/initialize.{cpp,c,cc}" --user "src/autonomous.{cpp,c,cc}" --user "include/main.{hpp,h,hh}" --user "Makefile" --target v5 --output bin/output.bin
 
 LIBV5RTS_EXTRACTION_DIR=$(BINDIR)/libv5rts
-$(LIBAR): $(call GETALLOBJ,$(EXCLUDE_SRCDIRS) $(EXCLUDE_FROM_LIB)) $(EXTRA_LIB_DEPS)
```

Comment:
Why is `EXCLUDE_SRCDIRS` getting removed?

Reply:
It's not quite being removed:
https://github.com/purduesigbots/pros/blob/f34a219cb49a6b5a8e81b6852abe4c84efa975d0/Makefile#L29-L32